climakitae/data/variable_descriptions.csv

```diff
@@ -58,4 +58,8 @@ tasmax,Statistical,daily,K,Daily maximum air temperature at 2m,The maximum daily
 tasmin,Statistical,daily,K,Daily minimum air temperature at 2m,The minimum daily air temperature at 2m above the Earth's surface. ,ae_orange,TRUE
 uas,Statistical,daily,m s-1,West-East component of Wind at 10m,"Wind coming from the west, measured at 10m above Earth's surface. By convention, wind coming from the east is negative.",ae_diverging,TRUE
 vas,Statistical,daily,m s-1,North-South component of Wind at 10m,"Wind coming from the north, measured at 10m above Earth's surface. By convention, wind coming from the south is negative.",ae_diverging,TRUE
-huss,Statistical,daily,kg/kg,Specific humidity at 2m,The percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.,ae_blue,TRUE
\ No newline at end of file
+huss,Statistical,daily,kg/kg,Specific humidity at 2m,The percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.,ae_blue,TRUE
+rel_humid_min,Statistical,daily,percent,Minimum relative humidity,The minimum percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.,ae_blue,TRUE
+rel_humid_max,Statistical,daily,percent,Maximum relative humidity,The maximum percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.,ae_blue,TRUE

```

Comment:
Making a note to discuss with the larger team during analytics planning

Reply:
I am keen to keep the variables as min/max rel hum (without "daily") since we will soon have monthly, and the code will require added complexity to deal with renaming that I'd prefer to avoid. RE: the descriptions, could I say something like this such that it would work for daily and monthly? "The minimum relative humidity computed for the selected temporal resolution ..."

Reply:
Yes, I think the 'daily' is implied if you've selected a daily time-resolution. 

And I suspect they are being requested for relevance to electricity demand (max) and fire risk (min) among others.

Reply:
Do we then remove "daily" from the min and max temperature variables, in the same line of thought? My comment was partially also related to consistency between our variable naming convention too

Reply:
Shoot-- I thought I had changed those variables (daily min/max temp) to say just min/max temp a while back. Yes, those should just say min/max temp as well